### PR TITLE
Remove, rather that just ls'ing, old logs

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: roles/syslog-aggregator/tasks/main.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@
   cron:
     name: log-cleanup
     user: root
-    job: "find {{ log_root }} -mtime +28 -type f -exec ls {} \\;"
+    job: "find {{ log_root }} -mtime +28 -type f -exec rm {} \\;"
```
Comment (seankelly) [inline roles/syslog-aggregator/tasks/main.yml]: Using `;` with `-exec` can be very slow. Fortunately, deleting is so common there is a `-delete` option.

Comment (djmitche) [inline roles/syslog-aggregator/tasks/main.yml]: Good point.  Is it the `;` that's so slow, or the `rm`?

Comment (seankelly) [inline roles/syslog-aggregator/tasks/main.yml]: I don't know; probably both. With `;` find executes the program for each argument while with `+` it uses as many arguments as possible for each execution. I've done some operations (I *think* not rm) with `;` that took hours but using `+` finished in seconds.

Comment (djmitche) [inline roles/syslog-aggregator/tasks/main.yml]: I see -- I wasn't even aware of the `+` option

**Pull request merged**